plugins/woocommerce/includes/admin/meta-boxes/views/html-order-item.php

```diff
@@ -104,7 +104,7 @@
 			?>
 		</div>
 		<?php
-			$step = $product->get_purchase_quantity_step();
+			$step = $product && $product->get_purchase_quantity_step();
```

Comment:
This will evaluate to bool. Maybe something like following

```suggestion
			$step = $product ? $product->get_purchase_quantity_step() : 1;
```